config/routes.rb

```diff
@@ -8,19 +8,22 @@
 
     # Patient routes
     # /patients/:id
-    # /patients/:id/calls 
+    # /patients/:id/calls
     # /patients/:id/notes
     # /patients/:id/external_pledges
-    resources :patients, 
+    resources :patients,
               only: [ :create, :edit, :update ] do
-      resources :calls, 
+      resources :calls,
                 only: [ :create, :destroy ]
-      resources :notes, 
+      resources :notes,
                 only: [ :create, :update ]
-      resources :external_pledges, 
+      resources :external_pledges,
                 only: [ :create, :update, :destroy ]
     end
 
+    get 'naf_status', to: 'clinics#naf_status', as: 'naf_status'
+    get 'clinics', to: 'clinics#index', as: 'clinics'
```

Comment:
I think we already have this route set as part of `resources :clinics` -- I think we can just use that. (We should keep `naf_status` though for sure!)